### DISCUSSION TITLE
Two backup drives....only one appears

### DISCUSSION THREAD

**Discussion by rifer1** [WinMerge/winmerge#1797] [General]
I've used this product for 3 years just fine.  Now I can't.  I have a backup of my Mac files, and of my PC files...I used to be able to compare them, but no longer: only the Mac drive is appearing.  Two screenshots are attached to help you understand my issue.  regards,
![Screenshot 2023-04-07 163940](https://user-images.githubusercontent.com/74878955/230678953-88009e49-43b4-466b-ad10-03107074663e.png)
![Screenshot 2023-04-07 163940](https://user-images.githubusercontent.com/74878955/230678991-dd1beb32-a478-4532-9eb1-c71790cb139d.png)
    R Rife

**Comment by sdottaka** [comment]
The two attached images appear to be identical.
Does this mean that this is not a WinMerge issue, but simply a problem with the drives not showing up?
